web/webhook_test.go

```diff
@@ -263,12 +261,13 @@ func TestCommandWebhooks(t *testing.T) {
 	assert.Equal(t, http.StatusBadRequest, resp.StatusCode)
 
 	for i := 0; i < 5; i++ {
-		if resp, appErr := http.Post(ApiClient.Url+"/hooks/commands/"+hook.Id, "application/json", bytes.NewBufferString(`{"text":"this is a test"}`)); err != nil || resp.StatusCode != http.StatusOK {
-			t.Fatal(appErr)
-		}
+		t.Run(fmt.Sprintf("hook #%d", i), func(t *testing.T) {
+			resp, err = http.Post(ApiClient.Url+"/hooks/commands/"+hook.Id, "application/json", bytes.NewBufferString(`{"text":"this is a test"}`))
+			assert.NoErrorf(t, err, "hook #%d failed", i)
```

Comment:
This line should use `require`, as if there is an error and if continues, the `resp` object can be `nil`:

```suggestion
			require.NoErrorf(t, err, "hook #%d failed", i)
```